docs-site/content/blog/frontend-website.md

```diff
@@ -119,7 +119,7 @@ $ pnpm dev
 
 First, move all our rest api endpoint under `/api` prefix. for doing it go to `src/app.rs`. in `routes` hooks function add `.prefix("/api")` to the default routes. 
 ```rust
-fn routes() -> AppRoutes {
+fn routes(_ctx: &loco_rs::prelude::AppContext) -> AppRoutes {
```

Comment:
Please update it to loco_rs::app::AppContext. Ensure that you apply this change consistently across all relevant sections in your modifications

Reply:
Will do! Now that I see the correct reference, I have modified to just &AppContext whenever &loco_rs::app::AppContext is already imported.